### PR TITLE
Initial CONTRIBUTING documentation

### PR DESCRIPTION
**Pull request by dragan**
- Useful for new comers and also for documenting steps we do not do very often

### CODE REVIEW

**Review by nicklathe — DISMISSED** [review]
Great! :shipit:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+Instead, this happened:
+
+<explanation>
+```
+
+These components are necessary in helping figure out what happened:
+
+1. What you did
+2. What you expected
+3. What happened instead.
```
Comment (ncronquist) [inline CONTRIBUTING.md]: Either remove the period here or add it to the lines above. I think it should probably be removed though.
```suggestion
3. What happened instead
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+
+[community]: https://github.com/scipian/community
+
+## Issue Triage
+
+We use [ZenHub][zenhub] within the team to triage and work on issues. Although 
+not required, we highly recommend you use it as well. Since, Scipian is a
```
Comment (ncronquist) [inline CONTRIBUTING.md]: No comma after since.
```suggestion
not required, we highly recommend you use it as well. Since Scipian is a 
```

**Review by ncronquist — DISMISSED** [review]
Looks good! :shipit:

File: CONTRIBUTING.md.tpl [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Contributing to Scipian
```
Comment (ncronquist) [inline CONTRIBUTING.md.tpl]: Will there be other repo templates like this? Do we want to store them all at the root of this repo or create a templates directory?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+- **Backlog**
+
+  Issues in this pipeline aren't the current focus. You'll usually see issues
+  here that have been triaged, but that are not necessarily complete yet and
+  haven't been assigned an iteration. Consider this pipeline part of our 
+  product backlog. Once a milestone is assigned, consider them part of the 
+  iteration back log.
+
+  For some issues, this may be the farthest it goes. Every week, we'll pick 
+  from the backlog the issues that add value to what we are doing. Yeah, most
```
Comment (nicklathe) [inline CONTRIBUTING.md:124]: Nit: `Yes` instead of `Yeah`?

File: CONTRIBUTING.md.tpl (hunk 3 above)
Comment (dragan) [reply CONTRIBUTING.md.tpl]: Good point, yes there will be others. I like the idea of creating a directory to hold them.

**Review by nicklathe — DISMISSED** [review]
I just have one suggestion, but it's not blocking this merge.

**Review by ncronquist — COMMENTED** [review]
Cool, looks good to me. I think you're good to squash your commits.

**Comment by nicklathe** [comment]
LGTM. Squash and rebase, thanks!

**Review by ncronquist — APPROVED** [review]
ship-it

**Pull request merged**